service/src/main/java/eu/europa/ec/fisheries/uvms/activity/service/mapper/FaCatchMapper.java

```diff
@@ -180,6 +184,60 @@ protected VesselTransportMeansEntity getVesselTransportMeansForRelocation(FaCatc
         return faCatch.getFishingActivity().getVesselTransportMeans().iterator().next();
     }
 
+    protected SizeDistribution getSizeDistribution(FaCatchEntity faCatch) {
+        if (faCatch == null) {
+            return null;
+        }
+
+        SizeDistribution result = new SizeDistribution();
+        if (faCatch.getSizeDistributionCategoryCode() != null && faCatch.getSizeDistributionCategoryCodeListId() != null) {
+            CodeType code = new CodeType();
+            code.setValue(faCatch.getSizeDistributionCategoryCode());
+            code.setListID(faCatch.getSizeDistributionCategoryCodeListId());
```

Comment:
Missing a row with result.setCategoryCodes ... code ... here, right?

Reply:
So embarrassing, you are completely right. Good thing you are a thorough reviewer!